### PR TITLE
Include openapi server code in pkg/connectors (modified from generated code)

### LINKED ISSUE #697: Include openapi server code in pkg/connectors (modified from generated code)
Need to include the openapi server code after modifying the generated code.

### PR DESCRIPTION
**Pull request by ritwikc**
This PR fixes #697 .

This is a follow up for PR #728 . This PR includes the following changes:

     (a) Updated Makefile in config/taxonomy with one time invocation of generate-server which generates the server code:
      Currently, the generated server code is in connector/open_policy_agent. Also included a file named model.go for imports. 

Signed-off-by: Ritwik <ritwik@datameshproj.sl.cloud9.ibm.com>

### CODE REVIEW

File: connectors/open_policy_agent/.openapi-generator-ignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
+# Generated by openapi-generator https://github.com/openapitools/openapi-generator
+
+# Use this file to prevent files from being overwritten by the generator.
+# The patterns follow closely to .gitignore or .dockerignore.
+
+# As an example, the C# client generator defines ApiClient.cs.
+# You can make changes and tell OpenAPI Generator to ignore just this file by uncommenting the following line:
+#ApiClient.cs
```
Comment (rohithdv) [inline connectors/open_policy_agent/.openapi-generator-ignore:9]: can remove this file

File: connectors/open_policy_agent/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM golang:1.10 AS build
+WORKDIR /go/src
+COPY go ./go
+COPY main.go .
+
+ENV CGO_ENABLED=0
+RUN go get -d -v ./...
+
+RUN go build -a -installsuffix cgo -o open_policy_agent .
```
Comment (rohithdv) [inline connectors/open_policy_agent/Dockerfile:9]: does this work. is this needed?

File: connectors/open_policy_agent/go/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Go API Server for open_policy_agent
+
+No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+
+## Overview
+This server was generated by the [openapi-generator]
+(https://openapi-generator.tech) project.
+By using the [OpenAPI-Spec](https://github.com/OAI/OpenAPI-Specification) from a remote server, you can easily generate a server stub.  
+-
+
```
Comment (rohithdv) [inline connectors/open_policy_agent/go/README.md:10]: can remove this.

File: connectors/open_policy_agent/.openapi-generator-ignore (hunk 1 above)
Comment (ritwikc) [reply connectors/open_policy_agent/.openapi-generator-ignore]: Done

File: connectors/open_policy_agent/Dockerfile (hunk 2 above)
Comment (ritwikc) [reply connectors/open_policy_agent/Dockerfile]: Removed the file.

**Review by roee88 — APPROVED** [review]

**Pull request merged**